application/management/commands/clear.py

```diff
@@ -0,0 +1,43 @@
+from itertools import chain
+import os
+import shutil
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from application.models import (MovieDirectory, Movie, MovieRequest, 
+                NewMovieNotification, WatchlistItem, Subtitle)
+
+
+class Command(BaseCommand):
+    help = "Clear all Cinema data (for testing purpose)"
+
+    def handle(self, *args, **options):
+        all_models = list(chain(MovieDirectory.objects.all(),
```

Comment:
Just my 2 cents, I would've gone with

```python
all_models = [MovieDirectory, Movie, ...]
for model in all_models:
    model.objects.delete()
```
It's been a while since I've touched django's ORM but I'm pretty sure that your  version here will query every single item out of the database and then delete it one by one instead of generating a nice `DELETE FROM whatever`

Reply:
To be honest, my use of Django and its ORM is probably far older than yours, so I won't argue against this. Plus, it's way more readable, which is always 👍

Reply:
I've read the Django doc, and this does not work. [It says:](https://docs.djangoproject.com/en/2.0/topics/db/queries/#deleting-objects)

> Note that delete() is the only QuerySet method that is not exposed on a Manager itself. This is a safety mechanism to prevent you from accidentally requesting Entry.objects.delete(), and deleting all the entries. If you do want to delete all the objects, then you have to explicitly request a complete query set:
> `Entry.objects.all().delete()`

So I can still make it more readable with your proposal, but with: `model.objects.all().delete()` in the loop. What do you think?

Reply:
Then maybe django doesn't execute the request before you try iterating on the results. I still think that doing all those `.all()` in the loop is cleaner